bootstrap.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+apt-get -y install python
```

Comment:
Is this really necessary? I thought python was installed by default in Debian and Ubuntu for ages, might be mistaken though.

Not a vagrant expert, might be stupid question: ansible is available automatically or is it executed by the host system and needs to be installed there?

Reply:
I got problems with provisioning without this pre-installation